src/edu/csus/ecs/pc2/ui/team/CreateTeamsTSV.java

```diff
@@ -0,0 +1,268 @@
+// Copyright (C) 1989-2021 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.ui.team;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.ParseArguments;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.exception.CommandLineErrorException;
+import edu.csus.ecs.pc2.core.imports.clics.TeamAccount;
+
+/**
+ * Read teams.json write teams.tsv
+ * 
+ * @author pc2@ecs.csus.edu
+ * @version $Id$
```

Comment:
I think this line is useless...

Reply:
There are many useless things, I have to assume that you meant it should be removed.
A statement is helpful - a specific requested change is better.

Reply:
Yes, of course I meant that it was useless _and should be removed_.  It's clearly something left over from The Good Old Daze...